src/ripple/shamap/SHAMap.h

```diff
@@ -150,6 +158,8 @@ class SHAMap
         Marked `const` because the data is not part of
         the map contents.
     */
+    void setFull ();
```

Comment:
Is the comment above on L158-160 meant for some other member function?

Reply:
Yes, will fix, thanks!